docs/tutorial/maps.rst

```diff
@@ -91,58 +69,84 @@ To get a quick look at your Map simply type:
-
-A number of other attributes are also available.
-For example, the `~sunpy.map.GenericMap.date`, `~sunpy.map.GenericMap.exposure_time`, `~sunpy.map.GenericMap.center` and others (see `~sunpy.map.GenericMap`).
-The full list can be found on `~sunpy.map.GenericMap`:
+This should show a representation of the data as well as some of its associated attributes.
+If you are in a Jupyter Notebook, this will show a rich HTML view of the table along with several quick-look plots.
+Otherwise, you can use the :meth:`~sunpy.map.GenericMap.quicklook` method to see these quick-look plots.
 
 .. code-block:: python
 
-    >>> map_date = my_map.date  # doctest: +REMOTE_DATA
-    >>> map_exptime = my_map.exposure_time  # doctest: +REMOTE_DATA
-    >>> map_center = my_map.center  # doctest: +REMOTE_DATA
+    >>> my_map.quicklook()  # doctest: +SKIP
 
-To get a list of all of the attributes check the documentation by typing:
+.. generate:: html
+    :html_border:
 
-.. code-block:: python
+    import sunpy.map
+    import sunpy.data.sample
+    my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+    print(my_map._repr_html_())
+
+.. _inspecting-maps:
+
+Inspecting Map Metadata
+=======================
 
-    >>> help(my_map)  # doctest: +SKIP
+The metadata for a Map is exposed via attributes on the Map.
+These attributes can be accessed by typing ``my_map.<attribute-name>``.
+For example, to access the date of the observation,
 
-Many attributes and functions of the map classes accept and return `~astropy.units.quantity.Quantity` or `~astropy.coordinates.SkyCoord` objects.
-Please refer to :ref:`units-sunpy` and :ref:`coordinates-sunpy` for more details.
+.. code-block:: python
+
+    >>> my_map.date  # doctest: +REMOTE_DATA
+    <Time object: scale='utc' format='isot' value=2011-06-07T06:33:02.770>
 
-The metadata for the map is accessed by:
+Notice that this is an `~astropy.time.Time` object which we discussed in the previous section :ref:`time-in-sunpy`.
+Similarly, we can access the exposure time of the image,
 
 .. code-block:: python
 
-    >>> header = my_map.meta  # doctest: +REMOTE_DATA
+    >>> my_map.exposure_time  # doctest: +REMOTE_DATA
+    <Quantity 0.234256 s>
 
-This references the metadata dictionary with the header information as read from the source file.
-To see if the metadata of a Map source has been modified, see :ref:`sphx_glr_generated_gallery_map_map_metadata_modification.py` for a demonstration.
+Notice that this returns an `~astropy.units.Quantity` object which we discussed in the previous section :ref:`units-sunpy`.
+The full list of attributes can be found in the reference documentation for `~sunpy.map.GenericMap`.
+These metadata attributes are all derived from the underlying FITS metadata, but are represented as rich Python objects, rather than simple strings or floating point numbers.
```

Comment:
```suggestion
These metadata attributes are all derived from the underlying FITS metadata, but are represented as rich Python objects, rather than simple strings or numbers.
```